pyproject.toml

```
@@ -0,0 +1,49 @@
+[tool.poetry]
+name = "pylint-odoo"
+version = "3.7.1"
+description = "Odoo plugin for Pylint"
+authors = ["Odoo Community Association (OCA) <support@odoo-community.org>"]
+license = "APGL3"
+homepage = "https://github.com/OCA/pylint-odoo"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python",
+]
+
+[tool.poetry.dependencies]
+python = ">=2.7,<3.0.0 || >=3.5.0"
+docutils = "0.16"
+lxml = "^4.6.3"
+polib = "1.1.0"
+restructuredtext-lint = "1.3.1"
+rfc3986 = "^1.4.0"
+six = "^1.15.0"
+whichcraft = "^0.6.1"
+pylint = [
+    {version = "^1.9.5", python = "^2.7"},
+    {version = "^2.6.0", python = "^3"}
+]
+Pygments = [
+    {version = "^2.2", python = "^2.7"},
+    {version = "^2.6.1", python = "^3"}
+]
+pylint-plugin-utils = "0.6"
```

Usually, when you're building a library (or a plugin, like here), it's better not to be so specific about requirements, if possible. This avoids conflicts such as #306.
```suggestion
python = ">=2.7,<3.0.0 || >=3.5.0"
docutils = "^0.16"
lxml = "^4.6.3"
polib = "^1.1.0"
restructuredtext-lint = "^1.3.1"
rfc3986 = "^1.4.0"
six = "^1.15.0"
whichcraft = "^0.6.1"
pylint = [
    {version = "^1.9.5", python = "^2.7"},
    {version = "^2.6.0", python = "^3"}
]
Pygments = [
    {version = "^2.2", python = "^2.7"},
    {version = "^2.6.1", python = "^3"}
]
pylint-plugin-utils = "^0.6"
```